docs/server/source/production-deployment-template/client-tls-certificate.rst

```diff
@@ -92,7 +92,7 @@ consolidated file containing both the public and private keys.
 
 .. code:: bash
 
-   cat /path/to/mdb-instance-0.crt /path/to/mdb-instance-0.key > mdb-instance-0.pem
+   cat /path/to/bdb-instance-0.crt /path/to/bdb-instance-0.key > bdb-instance-0.pem
```

Comment:
Why?
We never need a `bdb-instance-X` consolidated file.

Reply:
I wanted to update the `mdb` in the `client-tls-certificate` page because that is not relevant here. I know that is not correct, but in the `secret.yaml` file the secret corresponding to bdb certificate is named `bdb-instance.pem`, either we change that name to `bdb-instance.crt`, or keep this consolidated file.  There is no impact on the functionality. What do you suggest?

Reply:
:+1: